### PR TITLE
clean up setup file

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

* removed what appeared to be unreachable or unused code
* clean up batch file generation variable naming and comments

### Reviewer guidance

* Did I remove anything that is still being used?
* Should we open any issues to replace the 'todo' comments removed from the code?

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,66 +9,15 @@
 import os
 import sys
 
-try:  # for pip >= 10
-    from pip._internal.req import parse_requirements
-except ImportError:  # for pip <= 9.0.3
-    from pip.req import parse_requirements
```
Comment (benjaoming) [inline setup.py:14]: Have you researched the `pip` version on the various systems that we support?

**Review by benjaoming — APPROVED** [review]
Agreed, great changes and about time :) We seem to bundle dependencies as a default and the opposite hasn't been required. So the disabled code should go -- and if we need it again, it's over in KA Lite :+1:

File: setup.py (hunk 1 above)
Comment (benjaoming) [reply setup.py:14]: I think this will break `pip install kolibri` on a default Debian/Raspbian system.

https://packages.debian.org/stable/python-pip

Comment (indirectlylit) [reply setup.py:14]: where is `parse_requirements` used? the only place I saw was inside an `if False` block

Comment (rtibbles) [reply setup.py:14]: Yeah, it was added on the assumption that we would eventually do non-static pip builds for Kolibri, I guess?

Comment (benjaoming) [reply setup.py:14]: Oh right, didn't see it was made redundant further down, please continue! :+1:

**Comment by rtibbles** [comment]
Tried restarting the build to double check, but it's still failing. Given that you've edited setup.py, I assume the error is directly related to the PR, but I can't see where the issue is from a brief inspection of the code.

**Comment by benjaoming** [comment]
It's because we use `setup.py sdist --static`, and that option was our own, which we removed from `argv` before passing it on.. need to correct one or more build instructions, search for `--static`.

**Comment by jonboiser** [comment]
Buildkite is breaking with the `make dist` command. Does that need to be updated?

https://buildkite.com/learningequality/kolibri/builds/11265#92e32dd5-2a5d-4bbb-9a4f-5869265df3bb/549-2279

**Comment by indirectlylit** [comment]
sorry I haven't had a chance to come back to this. will take a look this week

**Pull request merged**